docs/FAQ.md

```diff
@@ -137,6 +137,17 @@ commit, then run `jj restore --from Y --to @-` to restore the parent commit
 to the old state, and `jj restore --from X` to restore the new working-copy
 commit to the new state.
 
+### How do I resume working on an existing change?
+
+There are two ways to resume working on an earlier change: `jj new` then `jj amend`,
+and `jj edit`. The first is recommended over the second for a few reasons, which
+come down to how intentional you are with your changes. If the change you want to
+resume working on has a conflict, `jj edit` would restore the state with plain-text
+conflict annotations. Editing those annotations incorrectly or by accident can lead
+to undefined behaviour during conflict resolution. If you amend a new change onto
```

Comment:
To mention UB here is quite a strange word choice. I'd rather use an actual example.

Reply:
A concrete example isn't something I can think of readily. I drafted that phrasing from @thoughtpolice's remarks on Discord
> it can propagate the "resolution" in an invalid manner down the series
I guess I could switch from "undefined behaviour" or "malformed deltas"?

Reply:
This would work in a highly technical context but I think it beneficial to users to simplify it a bit.

Reply:
I think it's well defined. I'd say something like "To keep the conflict-resolution work separate, it is recommended to resolve the conflicts in a new commit on top and then squash in the conflict resolutions."

Reply:
> I think it's well defined. I'd say something like "To keep the conflict-resolution work separate, it is recommended to resolve the conflicts in a new commit on top and then squash in the conflict resolutions."

My issue with using "to keep the conflict-resolution work separate" is that it assumes the user knows why they want that.

> This would work in a highly technical context but I think would be beneficial to users to simplify it a bit.

Then how about "undesired output from conflict resolution"?

Reply:
> My issue with using "to keep the conflict-resolution work separate" is that it assumes the user knows why they want that.

I meant it more as a recommendation. Maybe we can clarify that:

> We recommend keeping the conflict-resolution work separate from the existing changes in the conflicted commit. To do that, create a new commit on top of the conflicted commit and then squash in the conflict resolutions.

Reply:
> I meant it more as a recommendation.

I understand, I just didn't want to ignore the recommendation as I continued to iterate with Philip, so I shared my thoughts on it at the same time.

Reply:
I don't follow, but I can just take a look at the updated version when it's available.

Reply:
It's pushed